pkg/monitors/telegraf/monitors/winperfcounters/win_perf_counters_windows.go

```diff
@@ -31,7 +31,7 @@ func GetPlugin(conf *Config) (*telegrafPlugin.Win_PerfCounters, error) {
 
 	// Telegraf has a struct wrapper around time.Duration, but it's defined
 	// in an internal package which the gocomplier won't compile from
-	plugin.CountersRefreshInterval.Duration = conf.CountersRefreshInterval * time.Millisecond
+	plugin.CountersRefreshInterval.Duration = conf.CountersRefreshInterval.Get()
```

Comment:
@codesmith14 any idea why it was being multiplied by time.Millisecond here before?